content/docs/start/data-versioning.md

```diff
@@ -33,7 +33,7 @@ data source.
 </details>
 
 ```dvc
-$ dvc add data/data.xml
+$ dvc add data/data.xml data/.gitignore
```

Comment:
dvc add .gitignore ?

Nope, that's wrong, please revert. This line was fine.

Reply:
Done.

Reply:
Thanks!